gulpfile.js

```diff
@@ -91,7 +91,7 @@ function revUpdateReferences (config) {
     const rewrite = require('gulp-rev-rewrite')
     var manifest = gulp.src(path.join(config.dest, 'rev-manifest.json'))
 
-    return gulp.src(path.join(config.dest, '/**/**.{css,js}'))
```

Comment:
would `return gulp.src(path.join(config.dest, '**/**.{css,js}'))` work as well?

Reply:
No, it still does the same.